src/renderer/services/eth2/client/lighthouse/beacon.ts

```diff
@@ -17,18 +17,46 @@ import {ILighthouseSyncResponse} from "./types";
 import {parse as bigIntParse} from "json-bigint";
 import {Eth2ChainHeadType} from "../../../../models/types/head";
 import {IEth2ChainHead} from "../../../../models/head";
+import {ILogger} from "@chainsafe/lodestar-utils";
 
 
 export class LighthouseBeaconApiClient implements IEth2BeaconApi {
 
     private client: HttpClient;
-    private config: IBeaconConfig;
+    private readonly config: IBeaconConfig;
+    private readonly logger: ILogger;
 
     public constructor(options: IBeaconClientOptions) {
-        this.client = new HttpClient(options.baseUrl, {axios: {transformResponse: bigIntParse}});
+        this.client = new HttpClient(options.baseUrl, {
+            axios: {
+                headers: {
+                    "Content-Type": "application/json"
+                },
+                transformRequest: [
+                    (data: any): string => {
+                        //this will remove quotations around numbers
+                        try {
+                            return JSON.stringify(data).replace(/"([0-9]+\.{0,1}[0-9]*)"/g, "$1");
+                        } catch (e) {
+                            return data;
+                        }
+                    }
+                ],
+                transformResponse: [
+                    (data): any => {
+                        try {
+                            return bigIntParse(data);
+                        } catch (e) {
+                            return data;
+                        }
+                    }
+                ]
+            }
+        });
         this.config = options.config;
+        this.logger = options.logger;
```

Comment:
Should be some function to retrieve logger

Reply:
why?

Reply:
These ones that should be displayed as well to the user?

Reply:
but you are passing your implementation of this logger interface which pushes logs to stream